src/lib.rs

```diff
@@ -1,17 +1,83 @@
 // Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
 
-//! Client lib
+//! This crate provides an user-friendly client for [CeresDB](https://github.com/CeresDB/ceresdb).
+//!
+//! With this crate, you can access a standalone CeresDB or a CeresDB cluster
```

Comment:
you can access CeresDB(standalone or cluster)?

Reply:
Looks bad..